articles/sustainable-ai-with-tinyml/index.md

```diff
@@ -0,0 +1,155 @@
+
+![hero](/engineering-education/sustainable-ai-with-tinyml/hero.jpg)
+[Image source](https://images.unsplash.com/photo-1595865211152-eec48f9055c7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=762&q=80)
+
+Tiny machine learning is at the intersection of embedded devices and machine learning. It is being touted as "the next AI revolution". However, the carbon footprint of AI has been increasing with the evolution of AI. There is a need for more energy-efficient computing. This is one of the challenges TinyML can address.
+
+### Table of Contents
+
+1. Tiny machine learning and how it works
+
+2. Motivation for tiny machine learning
+
+3. Benefits of tiny machine learning
+
+4. Challenges to tiny machine learning
+
+5. Applications of tiny machine learning
+
+### Prerequisites
+
+Basic understanding of machine learning and embedded systems. For an introduction or refresher on some basic machine learning concepts check out this [article](https://www.section.io/engineering-education/supervised-learning-algorithms/).
+
+### Useful Terms
+
+**MCU** – [Microcontroller units](https://www.arrow.com/en/research-and-events/articles/engineering-basics-what-is-a-microcontroller) refer to integrated circuits used for specific applications such as in consumer electronics, telecommunications, and touch sensing solutions. They often control other parts of an electronic system and are able to interface with the physical world through built-in communication and peripherals.
+
+**IoT** – The internet of things is a system of computing devices, objects, digital machines and more, connected over the internet. These “things” have the ability to communicate over a network without requiring human interaction.
+
+**Embedded device** – An electronic device that uses a programmable device known as a microprocessor.
+
+**Embedded systems** – Computer systems with a dedicated function within a larger electrical or mechanical system.
+
+**Frugal objects** – Independent “things” which specialize in performing concrete operations that can be programmed with software as opposed to having specific electronics for each task.
```

Comment:
Give an example: I find that mentioning an example here will help understand the article at a later stage when this concept is used.